react/events/enhancedCommerce.ts

```diff
@@ -92,6 +93,40 @@ export const productImpression = (
   })
 }
 
+export const addToCart = (product: Product, price: number, quantity: number) => {
+  if (!product) return
+
+  ga('ec:addProduct', {
+    name: product.productName,
+    brand: product.brand,
+    variant: product.variant,
+    price,
+    quantity
+  });
+  ga('ec:setAction', 'add');
+  ga('send', 'event', {
```

Comment:
What is the purpose of nonInteraction here? 

The docs suggest this event: 
  `ga('send', 'event', 'UX', 'click', 'add to cart');     // Send data using an event.`